LICENSE.md

```diff
@@ -1,27 +1,9 @@
-Copyright 2016 Yahoo Inc.
```

Comment:
Should probably bring back the original here

Reply:
Ended up reverting the license back.